decidim-meetings/app/views/decidim/meetings/admin/meetings/index.html.erb

```diff
@@ -4,8 +4,8 @@
     <h2 class="item_show__header-title">
       <%= t(".title") %>
       <div class="button--title">
-        <%= link_to t("actions.new", scope: "decidim.meetings", name: t("models.meeting.name", scope: "decidim.meetings.admin")), new_meeting_path, class: "button button__sm button__secondary" if allowed_to? :create, :meeting %>
         <%= export_dropdown %>
+        <%= link_to t("actions.new", scope: "decidim.meetings"), new_meeting_path, class: "button button__sm button__secondary" if allowed_to? :create, :meeting %>
```

Comment:
Key consistency.
```suggestion
        <%= link_to t("actions.new_meeting", scope: "decidim.meetings"), new_meeting_path, class: "button button__sm button__secondary" if allowed_to? :create, :meeting %>
```

Reply:
Done at 3aec51d5923c74884fa49c1eed2b80219c5e7b3f
I prefer to do it on a new commit so we can add the key at the same time too 😄